manifests/spoke/spoke-clusterrolebinding.yaml

```diff
@@ -1,11 +1,13 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name: system:open-cluster-management:{{ .SpokeCoreName }}
+  name: system:open-cluster-management:{{ .SpokeCoreName }}-agent
```

Comment:
Can you update this name so that I know whether this is for the operator, registration, or work?  Sorry, I can't keep track at this point.

Reply:
i split the clusterroler/role for registration and work. So each agent would have different service account.  The clusterrole/clusterrolebinding for operator itself is not in this pr. It will be added in #9